toolkit/tools/internal/packagerepo/repocloner/rpmrepocloner/rpmrepocloner.go

```diff
@@ -270,10 +270,15 @@ func (r *RpmRepoCloner) Clone(cloneDeps bool, packagesToClone ...*pkgjson.Packag
 			pkgName,
 		}
 
-		if cloneDeps {
-			args = append([]string{"download", "--alldeps"}, args...)
-		} else {
+		// special case to skip dependencies for msopenjdk-11
+		if (pkgName == "msopenjdk-11-11.0.13+8_LTS-4.x86_64") || (pkgName == "msopenjdk-11-11.0.13+8-LTS-4.aarch64") {
```

Comment:
With the current solution any change to the version name will cause us to miss this check. How about:

```suggestion
		if strings.HasPrefix(pkgName, "msopenjdk-") {
```

Reply:
thanks